lib/fake_braintree/customer.rb

```diff
@@ -20,6 +20,10 @@ def create
       if invalid?
         response_for_invalid_card
       else
+        load_payment_method!(
+          @customer_hash.delete('payment_method_nonce'),
+          @customer_hash['credit_card'] ||= {}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.